ovos_plugin_manager/templates/tts.py

```diff
@@ -83,34 +142,76 @@ def clear_queue(self):
         except Exception:
             pass
 
+    def begin_audio(self):
+        """Perform beginning of speech actions."""
+        # This check will clear the "signal", in case it is still there for some reasons
+        check_for_signal("isSpeaking")
+        # this will create it again
+        create_signal("isSpeaking")
```

Comment:
Is it necessary to clear and recreate the signal here? It seems like extra FS checks

Reply:
not sure what the original idea was, just moved old code around.... i think the check can be removed

Reply:
Ah, maybe something to do with signal age?

Signals aren't properly handled as files, but with the SignalManager in Neon this would mean anything waiting for signal clear would return; file-based `wait_while_speaking` would only return if it happened to check in between lines 148 and line 150 execution (unlikely, but possible)